tlsconfig/certpool_go17.go

```diff
@@ -0,0 +1,15 @@
+// +build go1.7
+
+package tlsconfig
+
+import "crypto/x509"
+
+// SystemCertPool returns a copy of the system cert pool,
+// returns an empty cert pool if the system pool fails to load.
+func SystemCertPool() *x509.CertPool {
+	certpool, err := x509.SystemCertPool()
+	if err != nil {
+		return x509.NewCertPool()
```

Comment:
Wondering if this should error, otherwise it's essentially a silent failure, which could result in unexpected behaviour.